logger/options.go

```diff
@@ -8,9 +8,9 @@ import (
 	"time"
 )
 
-var (
+const (
 	defaultIngestURL     = "https://logs.logdna.com/logs/ingest"
-	defaultSendTimeout   = 5 * time.Second
+	defaultSendTimeout   = 30 * time.Second
```

Comment:
There's no tests that validate the default values, huh?

Reply:
there are tests that validate the defaults are applied - https://github.com/logdna/logdna-go/blob/aa78e9675d21a907d1ee7078c4aaae7d48411836/logger/options_test.go#L68

Reply:
ah, gotcha.  I wasn't sure how you could change it without fixing a test.  For things like this, I usually hardcode the values so it's explicit.